include/receiver_cadmium.hpp

```diff
@@ -67,8 +67,8 @@ class Receiver {
 
         // external transition
         void external_transition(
-		                TIME e,
-		                typename make_message_bags<input_ports>::type mbs) { 
+            TIME e,
+	        typename make_message_bags<input_ports>::type mbs) { 
             if (get_messages<typename defs::in>(mbs).size() > 1)
                 assert(false && "one message per time uniti");
```

Comment:
Curly braces are required even if there is only one statement. Hence if statement should look as below,
if (get_messages<typename defs::in>(mbs).size() > 1) {
    assert(false && "one message per time uniti");    
}

Reply:
Resolved.

Reply:
Please review the changes.